src/error.rs

```diff
@@ -345,10 +345,8 @@ impl Error {
     ///
     /// # Stability
     ///
-    /// Standard library backtraces are only available on the nightly channel.
-    /// Tracking issue: [rust-lang/rust#53487][tracking].
-    ///
-    /// On stable compilers, this function is only available if the crate's
+    /// Standard library backtraces are only available when using Rust ≥ 1.65.
+    /// On older compilers, this function is only available if the crate's
     /// "backtrace" feature is enabled, and will use the `backtrace` crate as
     /// the underlying backtrace implementation.
     ///
```

Comment:
A few lines down there is:

```rust
#[cfg_attr(doc_cfg, doc(cfg(any(nightly, feature = "backtrace"))))]
```

I don't know how to replace that :/
`any(>=1.65, feature = "backtrace")` surely won't work.

Reply:
I suppose an alternative would be bumping the MSRV? (And maybe bumping the Rust edition as well)